docker-compose.yml

```diff
@@ -17,15 +17,21 @@ services:
       VNC_SERVER_PASSWORD: ${VNC_SERVER_PASSWORD:-}
       TWOFA_TIMEOUT_ACTION: ${TWOFA_TIMEOUT_ACTION:-exit}
       AUTO_RESTART_TIME: ${AUTO_RESTART_TIME:-}
+      AUTO_LOGOFF_TIME: ${AUTO_LOGOFF_TIME}
+      SAVE_TWS_SETTINGS: ${SAVE_TWS_SETTINGS}
```

Comment:
The idea of implementing TWS in a container, instead of Gateway, is to allow the owner of Interactive Brokers account (for example your Client) to connect to. Let's imagine you created a trading bot application and you want to connect it to IB account of your Client. Your trading bot is running somehwere in the cloud and not in the same instance where TWS is running (because TWS is running on your Client cloud and you own the intellectual property of the trading bot). But how can your Client access to TWS and see the list of trades, or monitor what's going on? he can't open TWS on his computer since it will disconnect the TWS running in the Cloud (and your trading bot too). So he needs to connect remotely to the TWS in the cloud.
One problem with TWS is that there are settings which are not controlled by IBC and you need to apply them manually. Therefore, they need to be saved on IB servers in the Client profile.
From my staindpoint, the container will still be stateless because these settings are saved outside the container and into the Client profile.
Of course, this is my proposal but the final decision is up to you :-)

Reply:
I see your point, and I like the idea of having TWS as an option. Specially because we can use the same "IB Gateway" download to run ib-gateway or TWS by just switching the parameter on IBC start-up script.

As I say on my previous [reply](https://github.com/gnzsnz/ib-gateway-docker/pull/43#issuecomment-1817528069) there are thinks that can be committed and things that need further analysis. I think we can pass `TWS_SETTINGS_PATH` into things that can be committed. But please update it to `${SAVE_TWS_SETTINGS:-}`